lib/client-assets.php

```diff
@@ -375,6 +385,18 @@ function gutenberg_extend_wp_api_backbone_client() {
 				return model.prototype.route && route === model.prototype.route.index;
 			} ) );
 		};
+		wp.api.getTaxonomyModel = function( taxonomy ) {
+			var route = '/' + wpApiSettings.versionString + this.taxonomyRestBaseMapping[ taxonomy ] + '/(?P<id>[\\\\d]+)';
+			return _.first( _.filter( wp.api.models, function( model ) {
```

Comment:
I know this is copy-paste, but we should really use [`_.find`](http://underscorejs.org/#find) here so we're not filtering the entire list if match is one of the first entries.